LLama/Native/LLamaToken.cs

```diff
@@ -10,6 +10,9 @@ namespace LLama.Native;
 [DebuggerDisplay("{Value}")]
 public readonly record struct LLamaToken
 {
+    /// Token Value used when token is inherently null
+    public static readonly LLamaToken INVALID_TOKEN = -1;
```

Comment:
Correct naming for C# constants would be `InvalidToken`.

However, would it be better to replace this with a property like `IsInvalid => Value < 0`?

Reply:
When I said that I was assuming this was used for _detecting_ invalid tokens. I see below that's not true, so just renaming it is fine :)